your-project/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please move everything from `your-project/` into the root of your project directory. Then run `npm install` again.